modules/mcc/perf/perf_precomp.hpp

```diff
@@ -0,0 +1,20 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef __OPENCV_PERF_PRECOMP_HPP__
+#define __OPENCV_PERF_PRECOMP_HPP__
+
+#include "opencv2/ts.hpp"
+#include "opencv2/ts/cuda_test.hpp"
```

Comment:
Is that header required?

Reply:
1. The header `"opencv2/ts/cuda_test.hpp"` is needed to use `randomMat`
2. `#include "opencv2/ts/ts_perf.hpp"` instead `"opencv2/ts.hpp"` does not work

Reply:
Please use cv::randu instead